# -*- coding: utf-8 -*-
"""
default = 'Nothing'でエラーが出てるはずなので、そこを消してみる
元々は以下のような状態
field=models.ForeignKey(default='Nothing',on_delete=django.db.models.deletion.PROTECT, to='word_manage_app.Reference', verbose_name='参照')
"""


# Generated by Django 2.0.6 on 2018-11-30 01:19



from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        ('word_manage_app', '0009_auto_20181130_1009'),
    ]

    operations = [
        migrations.RemoveField(
            model_name='word',
            name='reference',
        ),
        migrations.AddField(
            model_name='word',
            name='reference',
            field=models.ForeignKey(on_delete=django.db.models.deletion.PROTECT, to='word_manage_app.Reference', verbose_name='参照'),
        ),
    ]
